modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -704,6 +704,77 @@ public void testClusterChangeTag() throws Exception {
         assertTrue("Tag has not been updated in 10 seconds", tagUpdated);
     }
 
+    /**
+     * Tests idle_verify working on an active cluster with persistence. Works via control.sh.
+     *
+     * @throws Exception if failed.
+     */
+    @Test
+    public void testIdleVerifyOnActiveClusterWithPersistence() throws Exception {
+        dataRegionConfiguration = new DataRegionConfiguration()
+                .setName("persistent-dataRegion")
+                .setPersistenceEnabled(true);
+
+        Ignite ignite = startGrids(1);
+
+        ignite.cluster().state(ACTIVE);
+
+        assertTrue(ignite.cluster().active());
+        assertEquals(ACTIVE, ignite.cluster().state());
+
+        ignite.createCache(new CacheConfiguration<>("persistent-cache")
```

Comment:
What purposes of this cache?

Reply:
Starting new cache with the given cache configuration

Reply:
IMHO, it is excessive too.

Reply:
I will delete the active test thus this line is removed as well